hybrid/layout/pv_layout.py

```diff
@@ -52,10 +52,10 @@ def __init__(self,
         self._system_model: Union[pv_simple.Pvwattsv8, pv_detailed.Pvsamv1] = solar_source
         self.min_spacing = min_spacing
 
-        self.module_power: float = module_power
-        self.module_width: float = module_width
-        self.module_height: float = module_height
-        self.modules_per_string: int = modules_per_string
+        self.module_power: float = get_module_power(self._system_model)
+        self.module_width: float = get_module_width(self._system_model)
+        self.module_height: float = get_module_length(self._system_model)
+        self.modules_per_string: int = get_modules_per_string(self._system_model)
```

Comment:
this seems less efficient than calling `get_module_attribs` and assigning from the dictionary

Reply:
`get_module_attribs` is specific to pvsamv1 where these functions provide parameters for either pvsamv1 or pvwatts, where the pvwatts parameters come from `hybrid.layout.pv_module`. You're right though--I've moved this functionality into `get_module_attribs`.